### DISCUSSION TITLE
Any guide for GIthub Apps on Github Enterprise?

### DISCUSSION THREAD

**Discussion by wichtounet** [community/community#24421] [Enterprise]
<p>Hi, </p>
<p>I am trying to follow the first guide to create a Github App: <a href="https://developer.github.com/apps/quickstart-guides/setting-up-your-development-environment/#authenticating-as-an-installation" rel="nofollow noopener">https://developer.github.com/apps/quickstart-guides/setting-up-your-development-environment/#authenticating-as-an-installation</a></p>
<p>Unfortunately, it does not contain any detail on how to work with Github Enterprise. </p>
<p>I have been able to figure out that I have to set api_endpoint when I create Octokit::Client, but then I am seeing other issues. </p>
<p>Is there any guide to help me do that?</p>
<p>Thanks</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>To my understanding, creating the app within the Enterprise instance and pointing to the instance’s API endpoint is what needs to happen. Can you share what other issues you’re running into?</p>

**Comment by wichtounet** [comment]
<p>Thanks lee-dohm. </p>
<p>It turned out our Github Enteprise endpoint was not the one I expected and the error message was more than cryptic. I figured out and now it works like a charm!</p>
